src/entity/user/User.ts

```diff
@@ -17,6 +18,7 @@ import type { UserStreakAction } from './UserStreakAction';
 import type { UserCompany } from '../UserCompany';
 import type { UserTopReader } from './UserTopReader';
 import type { SubscriptionCycles } from '../../paddle';
+import { Feed } from '../Feed';
```

Comment:
```suggestion
import type { Feed } from '../Feed';
```

Reply:
Really need to fix webstorm auto import type :jenkins: